src/open_inwoner/conf/locale/nl/LC_MESSAGES/django.po

```diff
@@ -2161,937 +1816,983 @@ msgstr ""
 msgstr "Geavanceerde opties"
 
-#: src/open_inwoner/configurations/admin.py:276
+#: configurations/admin.py:296
 msgid "Social media"
 msgstr "Sociale media"
 
-#: src/open_inwoner/configurations/admin.py:286
+#: configurations/admin.py:306
 msgid "Allowed CSS properties"
 msgstr "Toegestane CSS attributen"
 
-#: src/open_inwoner/configurations/admin.py:318
-#: src/open_inwoner/configurations/models.py:29
-#: src/open_inwoner/configurations/tests/test_colors.py:36
+#: configurations/admin.py:338 configurations/models.py:34
+#: configurations/tests/test_colors.py:36
 msgid "Primary color"
 msgstr "Primaire kleur"
 
-#: src/open_inwoner/configurations/admin.py:320
-#: src/open_inwoner/configurations/models.py:42
+#: configurations/admin.py:340 configurations/models.py:47
 msgid "Primary font color"
 msgstr "Primaire tekstkleur"
 
-#: src/open_inwoner/configurations/admin.py:325
-#: src/open_inwoner/configurations/models.py:34
-#: src/open_inwoner/configurations/tests/test_colors.py:40
+#: configurations/admin.py:345 configurations/models.py:39
+#: configurations/tests/test_colors.py:40
 msgid "Secondary color"
 msgstr "Secundaire kleur"
 
-#: src/open_inwoner/configurations/admin.py:327
-#: src/open_inwoner/configurations/models.py:49
+#: configurations/admin.py:347 configurations/models.py:54
 msgid "Secondary font color"
 msgstr "Secundaire tekstkleur"
 
-#: src/open_inwoner/configurations/admin.py:332
-#: src/open_inwoner/configurations/models.py:38
-#: src/open_inwoner/configurations/tests/test_colors.py:44
+#: configurations/admin.py:352 configurations/models.py:43
+#: configurations/tests/test_colors.py:44
 msgid "Accent color"
 msgstr "Accentkleur"
 
-#: src/open_inwoner/configurations/admin.py:334
-#: src/open_inwoner/configurations/models.py:56
+#: configurations/admin.py:354 configurations/models.py:61
 msgid "Accent font color"
 msgstr "Accent tekstkleur"
 
-#: src/open_inwoner/configurations/admin.py:339
+#: configurations/admin.py:359
 msgid "Warning banner background color"
 msgstr "De achtergrondkleur van de waarschuwingsbanner"
 
-#: src/open_inwoner/configurations/admin.py:341
+#: configurations/admin.py:361
 msgid "Warning banner font color"
 msgstr "Waarschuwingsbanner tekstkleur"
 
-#: src/open_inwoner/configurations/choices.py:6
+#: configurations/choices.py:6
 msgid "light"
 msgstr "licht"
 
-#: src/open_inwoner/configurations/choices.py:7
+#: configurations/choices.py:7
 msgid "dark"
 msgstr "donker"
 
-#: src/open_inwoner/configurations/choices.py:11
+#: configurations/choices.py:11
 msgid "Admin"
 msgstr "Hoofdbeheerder"
 
-#: src/open_inwoner/configurations/choices.py:12
+#: configurations/choices.py:12
 msgid "Regular user"
 msgstr "Standaardgebruiker"
 
-#: src/open_inwoner/configurations/models.py:26
+#: configurations/choices.py:16
+msgid "body_font_regular"
+msgstr ""
+
+#: configurations/choices.py:16
+msgid "Text body font"
+msgstr ""
+
+#: configurations/choices.py:17
+msgid "body_font_italic"
+msgstr ""
```

Comment:
Willen we hier geen NL vertalingen voor 'cursief' en 'vetgedrukt' etc?

Reply:
Nee, volgens mij wordt dit voor de bestandsnamen van de fonts gebruikt. Ik zal ze invullen maar gelijk houden